services/app-web/src/common-code/healthPlanFormDataType/healthPlanFormData.ts

```diff
@@ -228,16 +251,60 @@ const generateRateName = (
     return rateName
 }
 
+const removeRateRelatedDocuments = (
```

Comment:
I'm seeing some business logic in here - please add a unit test for this function.